CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Did this work? Apparently he CUDA language thing was added in 3.8:

check_language(CUDA)
if(CMAKE_CUDA_COMPILER)